docs/integrations/stripe.md

```diff
@@ -12,3 +12,171 @@ As Vue Storefront uses Nuxt.js under the hood and is built using Vue.js, you can
 ## Vendure
 
 To integrate Vendure with Stripe to accept and handle payments you can use official plugin that is described [here](https://www.vendure.io/docs/typescript-api/payments-plugin/stripe-plugin/)
+Once installed, the Vendure Stripe plugin will enable a new mutation exposed on the Vendure shop-api called createStripePaymentIntent.
+The high-level workflow is:
+
+    Create a “payment intent” on the server by executing the createStripePaymentIntent mutation.
+    Use the returned client secret to instantiate the Stripe Payment Element in VSF.
+    Once the form is submitted and Stripe processes the payment, the webhook takes care of updating the order without additional action in VSF.
+
+## Example of an integration using the nuxt-stripe-module <https://www.npmjs.com/package/nuxt-stripe-module>
+
+Before moving forward, be sure that both plugins (Vendure and vue-stripe or nuxt-stripe-module are correctly installed)
+
+### Step I: Extend the VSF API with a 'setStripePayment' method
+
+just call the Vendure's new mutation and you'll get a client secret in return.
+code example:
+
+    ```TS
+    import { CustomQuery } from '@vue-storefront/core';
+    import gql from 'graphql-tag';
+    import { Context } from '../../types';
+    import { NO_CACHE_FETCH_POLICY } from '../../helpers';
+
+    const setStripePayment = async (context: Context,customQuery?: CustomQuery) => { 
+    const request = await context.client.mutate({
+        mutation: gql`mutation {createStripePaymentIntent}`,
+        fetchPolicy: NO_CACHE_FETCH_POLICY
+    });  
+    return request.data;  
+    };
+    export default setStripePayment;
+    ```
```

Comment:
Could you please show the usage with this example as well (https://docs.vuestorefront.io/v2/integrate/extending-integrations.html#creating-an-extension)? I mean extendApiMethods in the middleware.config.js so that final users could easily get going with it? :)

Reply:
In the middleware.config.js you should be able to create a new extension with this method like following:

```ts
module.exports = {
  integrations: {
    vendure: {
      location: '@vue-storefront/vendure-api/server',
      configuration: {
        api: {
          uri: process.env.GRAPHQL_API,
          tokenMethod: process.env.TOKEN_METHOD
        }
      },
      extensions: (extensions) => [
         {
             name: 'stripe-extension',
             extendApiMethods: {
                setStripePayment: async (context) => {  // Your function here  }
             }
          }
       ]
    }
  }
};
```